src/main/java/org/openhab/binding/zwave/internal/protocol/commandclass/ZWaveClimateControlScheduleCommandClass.java

```diff
@@ -61,10 +72,80 @@ public void handleApplicationCommandRequest(SerialMessage serialMessage, int off
+            case SCHEDULE_CHANGED_GET:
+                logger.debug("Answering with noop SCHEDULE_CHANGED_REPORT");
+                getController().enqueue(getScheduleChangedReportMessage(SCHEDULE_CHANGE_TEMPORARILY_DISABLED));
+                break;
+            case SCHEDULE_OVERRIDE_REPORT:
+                OverrideType overrideType = getOverrideTypeFor(serialMessage.getMessagePayloadByte(offset + 1) & 0x03);
+                ScheduleState scheduleState = getScheduleStateFor((byte) serialMessage.getMessagePayloadByte(offset + 2));
+                logger.info("NODE {} reported: Override type: {}, ScheduleState: {}", this.getNode().getNodeId(), overrideType, scheduleState);
+                break;
             default:
                 logger.warn(String.format("NODE %d: Unsupported Command %d for command class %s (0x%02X).",
                         this.getNode().getNodeId(), command, this.getCommandClass().getLabel(),
                         this.getCommandClass().getKey()));
         }
     }
+
+    public ScheduleState getScheduleStateFor(byte messagePayloadByte) {
+        if (messagePayloadByte == 0x79) return new ScheduleState(ScheduleStateState.FROST_PROTECTION,0);
+        if (messagePayloadByte == 0x7A) return new ScheduleState(ScheduleStateState.ENERGY_SAVING, 0);
+        if (messagePayloadByte >= 0x7B && messagePayloadByte <= 0x7E) return new ScheduleState(ScheduleStateState.RESERVED, 0);
+        if (messagePayloadByte == 0x7F) return new ScheduleState(ScheduleStateState.UNUSED, 0);
+        return new ScheduleState(ScheduleStateState.SETBACK, messagePayloadByte);
+    }
+
+    public OverrideType getOverrideTypeFor(int i) {
+        switch (i) {
+            case 0b00: return OverrideType.NO_OVERRIDE;
+            case 0b01: return OverrideType.TEMPORARY_OVERRIDE;
+            case 0b10: return OverrideType.PERMANENT_OVERRIDE;
+            case 0b11: return OverrideType.RESERVED;
+            default: return OverrideType.RESERVED;
+        }
+    }
+
+    public SerialMessage getScheduleChangedReportMessage(byte scheduleChangeCounter) {
+        logger.debug("NODE {}: Creating new message for command SCHEDULE_CHANGED_REPORT", getNode().getNodeId());
+
+        SerialMessage result = new SerialMessage(getNode().getNodeId(), SerialMessage.SerialMessageClass.SendData,
+                SerialMessage.SerialMessageType.Request, SerialMessage.SerialMessageClass.SendData, SerialMessage.SerialMessagePriority.RealTime);
+
+        ByteArrayOutputStream outputData = new ByteArrayOutputStream();
+        outputData.write(getNode().getNodeId());
+        outputData.write(4);
+        outputData.write(getCommandClass().getKey());
+        outputData.write(SCHEDULE_CHANGED_REPORT);
+        outputData.write(scheduleChangeCounter);
+        result.setMessagePayload(outputData.toByteArray());
+        return result;
+    }
+
+    public enum OverrideType { NO_OVERRIDE, TEMPORARY_OVERRIDE, PERMANENT_OVERRIDE, RESERVED }
```

Comment:
I think it would be better to enhance the enum to include your getOverrideType method - this would be more consistent with the way such constants are handled elsewhere in the binding.